demos/tests/run_tests.py

```diff
@@ -153,6 +154,7 @@ def main():
         num_failures = 0
 
         os.putenv('PYTHONPATH',  "{}:{}/lib".format(os.environ['PYTHONPATH'], args.demo_build_dir))
+        os.environ['PYTHONIOENCODING'] = 'utf-8'
```

Comment:
You should be using `putenv` instead, because this variable should only be set for child processes. The _current_ process should respect the `PYTHONIOENCODING` that it was launched with.

Reply:
Done